Resources/Locale/en-US/store/store.ftl

```diff
@@ -2,6 +2,7 @@ store-ui-default-title = Store
 store-ui-default-withdraw-text = Withdraw
 store-ui-balance-display = {$currency}: {$amount}
 store-ui-price-display = {$amount} {$currency}
+store-ui-discount-display =  ({$amount}% off!)
```

Comment:
imo feels like this should be displayed after the tc cost and not before

`4 TC (50% off!)`

as opposed to `(50% off!) 4 TC`

Reply:
It will offset border of BUY buttons and make them non-aligned, that doesnt seem alluring. I will ask in ui channel of discord

Reply:
fixed

Reply:
fixed